.coveragerc

```diff
@@ -807,6 +807,7 @@ omit =
     homeassistant/components/nuki/sensor.py
     homeassistant/components/nx584/alarm_control_panel.py
     homeassistant/components/oasa_telematics/sensor.py
+    homeassistant/components/obihai/__init__.py
```

Comment:
I agree that this is annoying... I'm not sure why coverage didn't pick it up on the original PR.

Reply:
Whatever the issue is, it appears intermittently.  Saw it a few pushes to the Reboot Button PR yesterday as well.